Sources/Model/ObjC/Common/TracksEvent.m

```diff
@@ -254,6 +334,31 @@ - (BOOL)validPropertyName:(NSString *)propertyName
     return matches.count > 0;
 }
 
+- (BOOL)checkPropertyNameIsReserved:(NSString *)propertyName
+{
+    for (NSString *name in kReservedNames) {
+        if ([propertyName isEqualToString:name]) {
+            return YES;
+        }
+    }
+
+    return NO;
```

Comment:
I haven't verified it locally, but this might simplify the code:

```suggestion
    return [kReservedNames containsObject:propertyName];
```

Reply:
`containsObject` uses `isEqual` for testing equality and I wanted to use `isEqualToString`